packages/strapi-plugin-versioning/__tests__/versioning.test.ts

```diff
@@ -0,0 +1,77 @@
+const { isModelExists, findUid } = require("../src/services/Versioning");
+const strapiObject = require("../../strapi-global-object/strapi-global");
+
+describe("versioning test", () => {
+  test("isModelExists: should return false if model doesn't exists in strapi object", () => {
+    global.strapi = strapiObject;
+    const ctx = {
+      params: {
+        id: 1,
+      },
+      url: "/content-manager/content-types/application::test.test",
+    };
+
+    global.strapi.models = {
```

Comment:
why are You mutating global object (in rest of test it will be also modified)

Reply:
fixed